src/guides/v2.3/extension-dev-guide/framework/float-comparator.md

```diff
@@ -0,0 +1,38 @@
+---
+group: php-developer-guide
+title: Float Comparator
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+The [`Magento\Framework\Math\FloatComparator`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Math/FloatComparator.php){:target="\_blank"} library provides methods to compare float values with a proper epsilon value (`0.00001`).
+
+## Usage
+
+| Method               | Description                                                                                                                       |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `equal`              | Compares two given floats and returns a boolean value describing if they're equal.                                                |
+| `greaterThan`        | Compares two given floats and returns a boolean value describing if the first float is greater than the second float.             |
+| `greaterThanOrEqual` | Compares two given floats and returns a boolean value describing if the first float is greater than or equal to the second float. |
+
+## Examples
+
+### Check if two floats are equal
+
+```php
+    $isEqual = $this->floatComparator->equal(1.002, 1.002);
```

Comment:
Hi, @drpayyne !

What is `$this->floatComparator` ?

Thank you!

Reply:
Fixed, thanks!